admin/app/components/solidus_admin/base_component.rb

```diff
@@ -9,5 +9,9 @@ class BaseComponent < ViewComponent::Base
     def stimulus_id
       @stimulus_id ||= self.class.module_parent.to_s.underscore.dasherize.gsub(%r{/}, '--')
     end
+
+    def spree
```

Comment:
Did we evaluate adding

```
helper Spree::Core::Engine.routes.url_helpers
```

to admin base controller to access all the Solidus engine routes without being explicit about it?

Btw, it's so annoying to call this method `spree`, do you think we can find a valid alternative name if we keep it?

Reply:
> Did we evaluate adding`helper Spree::Core::Engine.routes.url_helpers` to admin base controller to access all the Solidus engine routes without being explicit about it?
> Btw, it's so annoying to call this method `spree`, do you think we can find a valid alternative name if we keep it?

We picked `spree` because that's the original engine name and matches the way you would call spree routes from any other engine or from the host application itself. Since we're not sure if all the routes between the `Spree::Backend` and `SolidusAdmin` will match I think it's better to be explicit. 

I think that mixing `Spree::Core` url helpers with those in `SolidusAdmin` would create more confusion and that is better to keep them clearly separated. We'll need the `spree` helper only while the two backend/admin are coexisting, once `SolidusAdmin` is complete we should be able to only use helpers coming `SolidusAdmin` routes.

In this case we're gonna need to link from `SolidusAdmin`'s `products#index` to `Spree::Backend` `products#edit`.

Does it make sense?

Reply:
Ok, so if I understand correctly, while we two backend/admin won't be coexisting anymore, we can deprecate this `spree` method and ask developers just to call the routes straight?

Reply:
Exactly, once we finish SolidusAdmin we won't need to link back to the old backend and we'll be able to eliminate that helper. Maybe we won't even need a deprecation because the new admin will be experimental while we use it.

Reply:
I guess it depends on how we want it to be used, my understanding was that the main nav would have a mix of new and old links on both sides, then if a URL is covered by the new admin we would have shown it.

Cross-linking makes usage seamless, at the cost of less UI/UX consistency. If we set expectations right I think we can go either way.

Thoughts?